tests/managed_services/mas_debezium/test_mas_debezium.py

```diff
@@ -0,0 +1,30 @@
+import shlex
+
+import pytest
+from constants import TEST_RECORD
+
+
+pytestmark = pytest.mark.mas_debezium
+
+
+class TestDebezium:
+    def test_kafka_topics(self, kafka_record, kafka_sa_acl, consumer_pod):
+        """
+        Test for managed kafka resources setup,
+        usage and teardown via rhoas sdk
```

Comment:
A test verifies one thing; this test verifies the topics in a consumer pod (and should be named as such); the other parts are done in fixtures, hence not part of the test but part of setup